packages/sdk/src/evm/constants/addresses.ts

```diff
@@ -50,6 +50,15 @@ export const CONTRACT_ADDRESSES: Record<
     twRegistry: TWRegistry_address,
     twBYOCRegistry: "0xB1Bd9d7942A250BA2Dce27DD601F2ED4211A60C4",
   },
+  // TODO
+  [ChainId.Sepolia]: {
```

Comment:
```suggestion
 11155111: {
```

let's hardcode this here instead of adding sepolia to our supported chain array which will break some stuff

Reply:
also no need for this right now if you don't add it to the supported chain array